stl.go

```diff
@@ -640,6 +653,22 @@ func (t *ttiBlock) bytes(g *gsiBlock) (o []byte) {
 	return
 }
 
+func (t *ttiBlock) bytesSTLToSTL(g *gsiBlock) (o []byte) {
```

Comment:
You shouldn't need this function after creating the `ItemMetadata` struct. You need to incorporate new logic into `newTTIBlock`.

Reply:
Done